controllers/pkg/reconcilers/approval/reconciler.go

```diff
@@ -123,17 +122,17 @@ func (r *reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 	}
 
 	if !pvReady {
-		r.recorder.Event(pr, corev1.EventTypeNormal,
-			"NotApproved", "owning PackageVariant not Ready")
+		r.recorder.Eventf(pr, corev1.EventTypeNormal,
+			"NotApproved", "owning PackageVariant for %s not Ready", pr.Spec.PackageName)
```

Comment:
Since this is already attached to `pr`, I don't think we need the name in the message. You see these events by looking at `kubectl describe packagerevision foo` , so we shouldn't need to repeat `foo` here.

Reply:
> I added these purely for additional logging detail. I found it difficult to debug with just the pr as it's just a long hex appended to the pr repo.

Ahh, right. Yeah, this change makes sense to me, then. Keep these here.